### DISCUSSION TITLE
Is getting a custom status really this hard?

### DISCUSSION THREAD

**Discussion by ChrisGallardo** [Rapptz/discord.py#5883] [Q&A] [answered]
So I've needed to get a user's custom status a couple times now for bots, and after some research I've found that this is how to do it:
```python
for act in member.activities:
    if isinstance(act, discord.CustomActivity):
        return str(act)
```
Is that really the easiest way? If so I think I'm going to file an issue and a pr.

**Comment by ChrisGallardo** [comment]
I added an issue for this #5884

**Comment by Rapptz** [comment, marked as answer]
Please don't crosspost like this next time. Thanks.
